src/core/ui.js

```diff
@@ -276,15 +276,14 @@ function rsErrorToHTML(err) {
     ? `<p class="respec-occurrences">Occurred <strong>${
         err.elements.length
       }</strong> times at:</p>
-    ${err.elements.map(generateMarkdownLink)}
-    `
+    ${markdownToHtml(err.elements.map(generateMarkdownLink).join(""))}`
     : "";
   const details = err.details
     ? `\n\n<details>\n${err.details}\n</details>\n`
     : "";
   const msg = markdownToHtml(`**${err.message}**`, { inline: true });
   const text = `${msg}${hint}${elements}${details}${plugin}`;
```

Comment:
Let's call this `html` then?

Reply:
called it result instead, as we use "html" for actual parsed html.